pkg/service/egress.go

```diff
@@ -368,7 +368,6 @@ func (s *EgressService) startWorker() error {
 				s.handleUpdate(info)
 			case <-s.shutdown:
 				_ = sub.Close()
-				rs.Stop()
```

Comment:
@frostbyte73 I had to remove this as it was panicking during test (closing of a closed channel)

It seems that during shutdown, this is ok? Especially since RedisStore is shared between a bunch of diff components.